tests/testthat/test-pool.R

```diff
@@ -120,9 +136,61 @@ test_that("pool", {
 
 })
 
+test_that("Pool (Rubin) works as expected when se = NA in analysis model", {
+    set.seed(101)
 
+    mu <- 0
+    sd <- 1
+    n <- 2000
+    vals <- rnorm(n, mu, sd)
+    real_mu <- mean(vals)
 
+    runanalysis <- function(x) {
+        list("p1" = list(est = mean(x), se = NA, df = NA))
+    }
 
+    results_bayes <- as_analysis(
+        method = method_bayes(n_samples = 5000),
+        results =
+            lapply(
+                seq_len(5000),
+                function(x) runanalysis(sample(vals, size = n, replace = TRUE))
+            )
+    )
+    bayes <- pool(results_bayes)
+
+    expect_equal(
+        bayes$pars$p1,
+        list(est = real_mu,
+             ci = as.numeric(c(NA, NA)),
+             se = as.numeric(NA),
+             pvalue = as.numeric(NA)),
+        tolerance = 1e-2
+    )
+
+    runanalysis <- function(x) {
+        list("p1" = list(est = mean(x), se = NA, df = Inf))
+    }
+
+    results_bayes <- as_analysis(
+        method = method_bayes(n_samples = 5000),
+        results =
+            lapply(
+                seq_len(5000),
+                function(x) runanalysis(sample(vals, size = n, replace = TRUE))
+            )
+    )
+    bayes <- pool(results_bayes)
+
```

Comment:
Might be worth running this with a few different options like alternative and conf.int just to make sure theres no interactions

Reply:
Ok!